Cargo.toml

```diff
@@ -90,6 +90,7 @@ engine_traits_tests = { path = "components/engine_traits_tests", default-feature
 error_code = { path = "components/error_code", default-features = false }
 fail = "0.5"
 file_system = { path = "components/file_system", default-features = false }
+flate2 = { version = "=1.0.11", default-features = false, features = ["zlib"] }
```

Comment:
Why `=`?

Reply:
I just copy from `tidb_query_expr` component. Changed to `1.0`.